nilearn/_utils/data_gen.py

```diff
@@ -193,78 +269,112 @@ def generate_labeled_regions(shape, n_regions, rand_gen=None, labels=None,
     return Nifti1Image(data, affine)
 
 
-def generate_labeled_regions_large(shape, n_regions, rand_gen=None,
-                                   affine=np.eye(4), dtype="int32"):
+def generate_labeled_regions_large(shape,
+                                   n_regions,
+                                   random_state=0,
+                                   affine=np.eye(4)):
```

Comment:
I might have missed something, but why was `dtype` removed?

Reply:
That should not have been removed. I will fix it.
I guess it's an issue of resolving a conflict from the original pull request.